### ISSUE TITLE
Make it easier to detach and delete volume

### ISSUE THREAD

**Issue by iver56** [hetznercloud/hcloud-python#61]
## Feature Request

**Is your feature request related to a problem? Please describe.**

In one scenario, when I run `volume.detach()`, I want to run `volume.delete()` afterwards. However, because the detach job is async in the background, my script has to wait for an unknown amount of time before it can successfully call `volume.delete()`

**Describe the solution you'd like**

Alternative 1: Make a new function that covers my use case: `volume.detach_and_delete()`

Alternative 2: Add an option to volume.detach that makes it block until the detach action is actually completed in the backend. It could look like this, for example:

`volume.detach(block_until_done=True)`

**Describe alternatives you've considered**

I could write a custom for loop like this in my script, but that's not code I want to have in my script. I would really prefer that the hcloud takes care of this.

```
def is_volume_detached_from_server(volume, server):
    # Implement code that gets the updated volume object or server object
    # and checks whether or not the given volume is still attached to the server
    return True | False

server = ...
volume.detach()

for i in range(50):
    if i == 49:
        raise Exception("Timeout")
    if is_volume_detached_from_server(volume, server):
        break
    else:
        time.sleep(5)
        continue
    

volume.delete()
```

**Teachability, Documentation, Adoption, Migration Strategy**

Migration strategy: None needed, since the requested change(s) would be backwards-compatible

**Comment by LKaemmerling** [comment]
We already have a function for waiting on the action success.

Sample:

```python
volume.detach().wait_until_finished()
```

https://hcloud-python.readthedocs.io/en/latest/api.clients.actions.html#hcloud.actions.client.BoundAction.wait_until_finished

So you don't have to implement this for yourself.

**Comment by iver56** [comment]
Thanks
